docs/source/DeveloperGuide/PilotsLogging/index.rst

```diff
@@ -0,0 +1,69 @@
+.. contents:: Table of contents
+   :depth: 3
+
+================================
+Pilots Logging system overview
+================================
+
+Pilots Loggins system is designed to allow logging of pilot state on every stage of lifecycle, including before installing
+DIRAC client and starting pilot process.
+
+Each logging entry includes:
+
+- current status of the Pilot - has to be one of predefined list of possible states,
```

Comment:
`possible states`: I understand this list of states is not in the current PR. 
When such definition is available (in another PR) please link to it.